.storybook/styles/docs.css

```diff
@@ -387,8 +387,8 @@
     top: 50%;
     left: 50%;
     border-radius: 50px;
-    border: 2px solid var(--o-ui-b-alias-low-break);
-    border-top-color: var(--o-ui-white);
+    border: 2px solid var(--hop-neutral-border-weak);
+    border-top-color: white;
```

Comment:
I suppose we don't have any white borders in the design system. I'd still put #fff though. May not be the case anymore, but some browsers could have different whites.

Reply:
Good catch.